playbooks/roles/create/tasks/main.yml

```diff
@@ -12,3 +12,12 @@
 # Clone the project_repo and sample_project_repo
 - name: "Clone the project repo {{ project_repo }}"
   import_tasks: clone_repos.yml
+
+- name: "Get approle creds"
```

Comment:
I think a more graceful solution would be to integrate pulling from Hashicorp with the `playbook_pre_tasks` role instead of hard-coding the path to the role in `{{ project_dir }}`.

For now maybe these two tasks should be run conditionally like: `when: {{ pull_hv_secrets|bool == true }}`

Reply:
The issue is that we need to clone the pipeline repo first in order to run this role and I don't think the role should be made public on GitHub.  I did mean to make the role conditional though, I'll add that in.

Reply:
All set